app/controllers/api/reviews/votes_controller.rb

```diff
@@ -1,10 +1,9 @@
 class Api::Reviews::VotesController < ApplicationController
-  before_action :authenticate_user!
+  before_action :authenticate_user!, only: :create
 
```

Comment:
If we do this, we have to change the index to check if there is an authenticated user and only get the vote if there is one.